client/src/hooks/useIncTypeOptions.js

```diff
@@ -17,42 +17,43 @@ export function useIncTypeOptions() {
 
   const [incTypesOptions, setIncTypesOptions] = useState();
 
-  const covidKeywords = ["covid19","covid","coronavirus"];
-
-  const covidException = (desc, keywords) => {
+  const checkCovidSpecialCase = (desc, keywords) => {
     const lowercasedDescription = desc.toLowerCase();
     return (
-      keywords.find((keyword) =>
-          lowercasedDescription.includes(keyword.toLowerCase()
-          )
+      keywords.find((keyword) => lowercasedDescription.includes(keyword.toLowerCase())
       ) || null
     );
   };
 
   const updateOptionsFromDescription = useCallback(
     async (description, options) => {
-       // specialcase to be handled for Covid keywords
+      // specialcase to be handled for Covid keywords
+      const covidKeywords = ["covid", "coronavirus"];
 
       if (options) {
-        if(covidException(description,covidKeywords)!= null){
-        
-          setIncTypesOptions(Object.values(options));
-          setIncidentTypePriAutocomplete({label: "COVID-19 Confirmed", 
-                                          value: "COVID-19 Confirmed", 
-                                          confidence: "1"});
-         
-          }
-        else{
-        const { updatedIncTypes, topIncType } = await getMultiPrediction(
-          description,
-          options
+        const covidOptionExist = options.filter((type) =>
```

Comment:
you can just find `options.find` here. It'll return the option if it exists and `undefined` if it doesn't, so we won't need to pull it off of the array. Then just rename this to `covidOptionIfExists`.